README.md

```diff
@@ -40,6 +40,11 @@ sequence = [
     rate = 4
     duration = 1
   }
+  { type = cycle        # cycle through 'values' periodically
```

Comment:
I don't think `periodically` makes sense here. It means 'at regular interval'. 

`cycle though the 'value'` should be good enough.